src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/NoProvidersAlert.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export default function NoProvidersAlert({ currentLocation, typeOfCareName }) {
+  const headline = `We can’t find any ${typeOfCareName} providers close to you`;
+  return (
+    <AlertBox
+      status="info"
+      headline={headline}
+      className="vads-u-margin-top--3"
+      content={
+        <>
+          <p>
+            We can't find providers within 60 miles from your{' '}
```

Comment:
"can't" should be using right single quote.

Reply:
Good catch, fixed.